src/Exceptions/HandlerThrowable.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Anik\ElasticApm\Exceptions;
+
+use Anik\ElasticApm\Spans\ErrorSpan;
+use Exception;
+use Illuminate\Contracts\Debug\ExceptionHandler;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Throwable;
+
+class HandlerThrowable implements ExceptionHandler
+{
+    private $primaryHandler;
+    private $ignoredExceptions = [
+        NotFoundHttpException::class,
+    ];
+
+    public function __construct (ExceptionHandler $primaryHandler, array $ignoredExceptions = []) {
+        $this->primaryHandler = $primaryHandler;
+
+        if ($ignoredExceptions) {
+            $this->ignoredExceptions = $ignoredExceptions;
+        }
+    }
+
+    private function logException (Exception $e) {
+        $depth = config('elastic-apm.error.trace_depth', 30);
+        $traces = collect($e->getTrace())->take($depth)->map(function ($trace) {
+            return [
+                'file/func' => $trace['file'] ?? ($trace['function'] ?? 'N/A'),
+                'line'      => $trace['line'] ?? 'N/A',
+            ];
+        });
+        app('apm-agent')->addSpan(new ErrorSpan($e, $traces));
```

Comment:
Also, please change the `ErrorSpan`'s first parameter from `Exception` to `Throwable`

Reply:
> I'm not very happy to create an Exception object for Span, but so our changes are minimal so as not to cause a breaking change. However, now I realized that we lost traceability, I will analyze this situation with more care...

I actually meant to change the `ErrorSpan`'s `__construct` parameter instead of creating a new error object.
```
# in ErrorSpan
public function __construct(\Throwable $t, Collection $traces) {
// ...
}
```
`Throwable` is an interface, `Exception` implements that interface. Should not break the existing code.